src/pages/index.js

```diff
@@ -59,57 +59,119 @@ const cardSection = new Section(
   selectors.cardSection
 );
 
+// Popup instances
 const imagePopup = new PopupWithImage(selectors.previewImagePopup);
+const profileEditPopup = new PopupWithForm(
+  selectors.profileEditPopup,
+  (formData) => {
+    return api
+      .updateUserInfo({
+        name: formData.title,
+        about: formData.description,
+      })
+      .then((updatedData) => {
+        userInfo.setUserInfo({
+          name: updatedData.name,
+          job: updatedData.about,
+        });
+        profileEditPopup.close();
```

Comment:
`        profileEditPopup.close();
`

You close the modal inside `PopupWithForm`, so you don't need it in `index.js`

**Please, correct it everywhere in the code**

Reply:
Ok. Thank you.